src/js/constants.js

```diff
@@ -1,3 +1,2 @@
-const API_ROOT = process.env.BACKEND_SERVER;
-
-export { API_ROOT };
+export const API_ROOT = process.env.BACKEND_SERVER;
+export const DEFAULT_IMAGE = "/static/images/shop/placeholder.gif";
```

Comment:
not a fan of the hardcoded `/static/` for static URL here - can we pull that from the env as well instead?

Reply:
Ah yeah, i was wondering how we'd get that fro back-end, same for product/category detail pages.